introduction.md

```diff
@@ -0,0 +1 @@
+Hello, this is introduction of Sergey Shihov to complete the codeschool course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.